### PR TITLE
feat(ui): Remove formatting on paste into smart search bar

### PR DESCRIPTION
**Pull request by scttcper**
Fixes an issue with text copied with formatting where it would always leave an extra space at the beginning

https://getsentry.atlassian.net/browse/ISSUE-1352

### CODE REVIEW

File: static/app/components/smartSearchBar/index.tsx [hunk 1]
```diff
@@ -464,6 +464,22 @@ class SmartSearchBar extends React.Component<Props, State> {
...
+    evt.preventDefault();
+
+    // get text representation of clipboard
+    const text = evt.clipboardData.getData('text/plain').replace('\n', '').trim();
+
+    // insert text manually
+    const query = this.state.query + text;
+    this.setState(makeQueryState(query), this.updateAutoCompleteItems);
+    callIfFunction(this.props.onChange, query, evt);
+  };
```
Comment (evanpurkhiser) [inline static/app/components/smartSearchBar/index.tsx:481]: What happens when I paste at the front, or middle?

File: static/app/components/smartSearchBar/index.tsx [hunk 2]
```diff
@@ -464,6 +464,35 @@ class SmartSearchBar extends React.Component<Props, State> {
...
+    // Get text representation of clipboard
+    const text = evt.clipboardData.getData('text/plain').replace('\n', '').trim();
+
+    // Create new query
+    const currentQuery = this.state.query;
+    const cursorPosStart = this.searchInput.current!.selectionStart;
+    const cursorPosEnd = this.searchInput.current!.selectionEnd;
+    const textBefore = currentQuery.substring(0, cursorPosStart);
+    const textAfter = currentQuery.substring(cursorPosEnd, currentQuery.length);
+    const mergedText = textBefore + text + textAfter;
```
Comment (priscilawebdev) [inline static/app/components/smartSearchBar/index.tsx:483]:
```suggestion
    const mergedText = `${textBefore}${text}${textAfter}`;
```

File: static/app/components/smartSearchBar/index.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/components/smartSearchBar/index.tsx:494]: we fixed this

**Review by priscilawebdev — APPROVED** [review]
code looks good to me

**Pull request merged**